content/docs/pipelines/sdk/output-viewer.md

```diff
@@ -147,14 +147,21 @@ viewers later on the page.
           which case the Kubeflow Pipelines UI concatenates the data from the 
           matching source files.</p>
           
-          <p>For some viewers, this field can contain inlined string 
-          data instead of a path.</p>
+          <p><code>source</code> can also contain inlined string data instead of
+          a path when <code>storage='inline'</code>.</p>
           </td>
       </tr>
       <tr>
         <td><code>storage</code></td>
-        <td>Applies only to outputs of type <code>markdown</code>. See 
-        <a href="#type-markdown">below</a>.</td>
+        <td><p>Optional. When <code>storage</code> is 'inline', value of
+        <code>source</code> is parsed as inline data instead of a path. Applies
```

Comment:
Applies to all types of outputs except <code>tensorboard</code>.

I'd recommend revising this to:

This applies to all types of outputs except <code>tensorboard</code>.


Question, what applies to all types of outputs? That storage=inline indicates that source contains inline data? If so, what happens if storage=inline for tensorboard?

Reply:
Thanks, changed.

For the question, tensorboard simply ignores this config because tensorboard is a separate server, it can only read from external source.